regression_test_wrapper.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Should we remove this script prior to merge?  We had always intended to remove it, but it may prove useful until for verifying changes until the v1 wrapper is dropped in v2.0.0.